### ISSUE TITLE
How to set x axis labels at fixed interval

### ISSUE THREAD

**Issue by sanjay4one** [PhilJay/MPAndroidChart#3000]
I am displaying realtime chart which should display values as per second. My X axis is time in second. But I am not able to display fixed time interval in x axis ie. 0,1,2,... so on. X axis value is automatically calculated and time interval between two x values goes to 20 seconds which I don't want. I need your help to fix this.

Any help would be greatly appreciated.

**Comment by RKGG** [comment]
Do you want to set a fixed interval in x axis? I set my linechart with a fixed interval in x axis by the follow code.
`ArrayList<String> xValues = new ArrayList<String>();
        for (int i = WLstartpoint; i <= WLendpoint; i+=WLspace) {
            xValues.add("" + i);
        }`
The WLspace is my fixed interval. I hope it can help you.

**Comment by JCLaHoot** [comment]
did this work? I have a similar issue where I need to set the interval for my labels...

**Comment by RKGG** [comment]
Yes. It worked for me. It just uses this code " i+=WLSpace" . And you need to kown where is the starting and ending of x axis firstly  so you can difine the fixed interval what you want. I hope it can help you!
